public/app/features/logs/components/getLogRowStyles.ts

```diff
@@ -70,6 +70,7 @@ export const getLogRowStyles = memoizeOne((theme: GrafanaTheme2) => {
       font-family: ${theme.typography.fontFamilyMonospace};
       font-size: ${theme.typography.bodySmall.fontSize};
       width: 100%;
+      margin-bottom: ${theme.spacing(2.25)};
```

Comment:
Could we add a comment here what is the purpose of this styling - so that the context button for last row is displayed . And when/if we decide to change the position of show context/copy line button, this will not be needed anymore.